src/server/common/forms/definitions/example-grant-with-auth.yaml

```diff
@@ -0,0 +1,549 @@
+          Selecting 'No' will demonstrate navigating to a Terminal page to end the application
+        type: YesNoField
+        options:
+          customValidationMessages:
+            any.required: Select 'Yes' to continue
+  - title: You cannot apply for a grant from this scheme
+    path: '/cannot-apply'
+    controller: TerminalPageController
+    components:
+      - name: cannotApplyInfo
+        title: Html
+        type: Html
+        content: |-
+          <p class="govuk-body">This is an example of a Terminal page to give details on what conditions have not been met to continue applying for the grant.</p>
+    condition: terminalCondition
+
+  - title: Which country is your business based in?
+    path: '/country'
+    components:
+      - name: supportedCountries
+        title: Supported Countries
+        type: List
+        list: countryList
+      - name: country
+        title: Business country
+        type: AutocompleteField
+        list: countryList
+        hint: Start typing to see a filtered list of countries to select from
+        options:
+          customValidationMessages:
+            any.required: Enter a country
+
+  - title: What is your business?
+    path: '/nature-of-business'
+    components:
+      - name: businessNature
+        title: What is your business?
+        type: RadiosField
+        list: natureOfBusinessList
+        hint: Selecting the first option demonstrates navigating to an optional page before continuing
+        options:
+          customValidationMessages:
+            any.required: Select the option that applies to your business
+
+  - title: You may be able to apply for a grant from this scheme
+    path: '/nature-of-business-may-apply'
+    components:
+      - name: natureOfBusinessMayApplyInfo
+        title: Html
+        type: Html
+        content:
+          '<p class="govuk-body">You must meet certain conditions before
+          you submit a full application. The application deadline is 31 May 2025</p>
+
+          '
+      - name: natureOfBusinessMayApplyLink
+        title: Html
+        type: Html
+        content:
+          "<p class='govuk-body'><a href='https://www.gov.uk/guidance/funding-for-farmers'
+          class='govuk-link govuk-link--no-visited-state'>See other grants you may be
+          eligible for</a></p>"
+    condition: choiceOneCondition
+
+  - title: What eligible items does your project need?
+    path: '/project-items'
+    components:
+      - name: eligibleItems
+        title: What eligible items does your project need?
+        hint: |
+          Use checkboxes to allow selection of one or more options.
+          This example requires at least one selection, but the component can be set to optional.
+        type: CheckboxesField
+        list: projectItemsList
+        options:
+          customValidationMessages:
+            any.required: Select all the items your project needs for now
+
+  - title: What is the estimated cost of the items?
```

Comment:
This is over-specific for this journey. We don;t want calculations, or specifics like VAT / costs. Just how a number entry works.